andela/bp-esa-frontend
# #167296170 Redirect to AIS when fellow name is clicked in card view

#### What does this PR do?
- This PR adds functionality to Redirect to AIS when a fellow's name is clicked in card view

#### Description of Task to be completed?
- Redirect to AIS when a fellow's name is clicked in Card view
- Add a tooltip to the fellow name in the table view
- Add a tooltip to the fellow name in the card view

#### How should this be manually tested?
1. Git clone the repository.
2. Checkout to the `ft-ais-link-in-card-view-167296170` branch.
5. Start the server `npm run start` _(or the `yarn` equivalent)_.
6. Clone the backend repository, `https://github.com/andela/bp-esa-backend`.
7. Checkout to the backend `develop` branch.
8. Start the backend server `npm run start:dev` _(or the `yarn` equivalent)_.
9. Login to the ESA application
10. Click the card view icon.
11. Hover over a fellow's name in a card
A tooltip with an external link icon should appear
13. Click the fellow's name
You should be redirected to the fellow's AIS page.
14. Click the table/list view icon.
15. Hover over a fellow's name
A tooltip with an external link icon should appear

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[167296170](https://www.pivotaltracker.com/story/show/167296170)

#### Screenshots (If applicable)
##### List view redirect icon
<img width="1408" alt="Screenshot 2019-07-18 at 19 54 00" src="https://user-images.githubusercontent.com/38077368/61476472-d2996400-a995-11e9-8bf1-9a888b7b7403.png">


##### Card view redirect icon
<img width="271" alt="Screenshot 2019-07-18 at 19 53 05" src="https://user-images.githubusercontent.com/38077368/61476536-ee046f00-a995-11e9-983c-34b7acff3e94.png">
